C/CMake/build_tarballs.jl

```diff
@@ -3,17 +3,17 @@
 using BinaryBuilder
 
 name = "CMake"
-version = v"3.23.3"
+version = v"3.25.2"
 
 # Collection of sources required to build CMake
 sources = [
-    ArchiveSource("https://github.com/Kitware/CMake/releases/download/v$(version)/cmake-$(version).tar.gz",
-                  "06fefaf0ad94989724b56f733093c2623f6f84356e5beb955957f9ce3ee28809"),
```

Comment:
Oh OK. Do you want me to change it back? Personally I like the GitSources, because I can just copy&paste the SHA from the webpage without having to download the archive to determine the hash. But yeah I did change this with the recent issues in mind.

Reply:
No, it's fine using the GitSource.  Now BinaryBuilder barks at you if you use the unstable URL: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/293